src/ColorVectorSpace.jl

```diff
@@ -32,9 +32,7 @@ import Base:      conj, sin, cos, tan, sinh, cosh, tanh,
 
 export dotc
 
-AbstractGray{T} = Color{T,1}
-TransparentRGB{C<:AbstractRGB,T}   = TransparentColor{C,T,4}
-TransparentGray{C<:AbstractGray,T} = TransparentColor{C,T,2}
+# TODO: we get rid of these definitions or move them to ColorTypes.jl
```

Comment:
Just a comment, we don't have to move *all* "typealiases" to ColorTypes, just the ones that are likely to be used again and again. It's not so obvious that the `Float` variants are widely useful, but they are useful to this package.